src/arguments/utility.py

```diff
@@ -33,13 +34,122 @@
 
 console = Console()
 
+class Playbook():
+    def __init__(self):
+        self.linux_path = "/home/{}/Documents/dynamic".format(os.getenv('USER'))
+        self.mac_path = "/Users/{}/Documents/dynamic".format(os.getenv('USER'))
+        self.file_name = 'dynamic_playbook.json'
```

Comment:
What if we can create an environment variable and store it in the file
```python
import os

file = "example_name.json"
key = "DYNAMIC"

def get_playbook_file(key):
    data = os.getenv(key)
    if not data:
        data = filename
         os.environ[key] = filename
     return data
```

What do you say @GouravSardana and @NamamiShanker

Reply:
@pranavbaburaj, you mean an environment variable will store the playbook questions and answers? We will make changes to it and then store it in the file?

Reply:
No, the environment variable will store the file in which the questions and answers are stored.

Reply:
You mean the path to file?

Reply:
Yeah

Reply:
Normally I have only seen API keys, authentication files, and path to important scripts stored in the environment. I want to have your view on how storing its path in the environment will help.

Reply:
I agreed because later on we can use these path somewhere else as well (if needed).

Reply:
Okay then I'll add it